tests/models/edx/converters/xapi/test_base.py

```diff
@@ -0,0 +1,19 @@
+"""Tests for the base xAPI Converter"""
+
+from ralph.models.edx.converters.xapi.base import BaseXapiConverter
+
+
+class DummyBaseXapiConverter(BaseXapiConverter):
```

Comment:
I would integrate this  class in the `test_models_edx_converters_xapi_base_base_xapi_converter_successful_initialization` function

Reply:
Good Idea) Thanks) Moved the class definitions inside the test methods)